src/components/Navbar/index.view.tsx

```diff
@@ -1,8 +1,37 @@
 import React from "react";
 import "./Navbar.scss";
 
-const Navbar: React.FC = () => {
-  return <div className="Navbar">Hello from Navbar</div>;
+interface NavbarProps {
+  title: string;
+  sponsorURL: string;
+  hackerURL: string;
+}
+
+const Navbar: React.FC<NavbarProps> = ({
+  title,
+  sponsorURL,
+  hackerURL,
+}: NavbarProps) => {
+  return (
+    <div className="Navbar">
+      <div className="Navbar__title">{title}</div>
+      <div className="Navbar__buttonContainer">
+        <a href={sponsorURL} target="_blank" rel="noreferrer">
+          <button
+            className="Navbar__button"
+            aria-label="sponsor sign-up button"
+          >
+            Be a sponsor!
+          </button>
+        </a>
+        <a href={hackerURL} target="_blank" rel="noreferrer">
```

Comment:
Perhaps we can make a Button component instead of using `<a href="xxx" target="_blank" rel="noreferrer">` each time, if we are going to have many links.